ivy/functional/backends/tensorflow/experimental/linear_algebra.py

```diff
@@ -148,3 +148,20 @@ def multi_dot(
         raise ValueError("Expecting at least two tensors.")
     dot_out = reduce(tf.matmul, x)
     return dot_out
+
+
+def solve_traingular(
+    a: Union[tf.Tensor, tf.Variable],
+    b: Union[tf.Tensor, tf.Variable],
+    /,
+    *,
+    lower: bool = False,
+    adjoint: bool = False,
+    out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+) -> Union[tf.Tensor, tf.Variable]:
+    return tf.linalg.triangular_solve(
```

Comment:
no `unit_diagonal` argument here.  Pay close attention to the superset behavior, and which framework you're adhering to for the same